app/components/skip_link_component.html.erb

```diff
@@ -1,5 +1,5 @@
-<nav id="skip" class="visually-hidden-focusable" aria-label="<%= t('blacklight.skip_links.label') %>">
+<nav id="skip-link" class="visually-hidden-focusable" aria-label="<%= t('blacklight.skip_links.label') %>">
```

Comment:
This was intentionally made to not be `#skip-link` so that we didn't have to fight against the CSS in blacklight.

Reply:
The CSS for #skip-link is coming in from spotlight, with really just one change required for the position to make it appear correctly

Reply:
Right, but we want all of the dlss sites to have the same skip link look, even if they aren't using Spotlight.